tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -67,6 +68,21 @@ export const compileProgram =
       return pipeline;
     };
 
+export const typeSnippet = (component: number, type = 'f32') => {
```

Comment:
The parameter orders of typeSnippet and dataTypeToGPUType are different. We may also refactor this part in future.